apps/sms/views/conversation/style/message.css

```diff
@@ -142,6 +142,20 @@
   color: #d1eeee;
 }
 
+.message.incoming .message-details .late-arrival-notice {
+  color: #026a69;
+  font-size: 1.3rem;
+  display: flex;
+  align-items: center;
+}
+
+.message.incoming .message-details .late-arrival-notice:before {
+  content: url('images/icons/late_clock.png');
```

Comment:
I think we'll want to use a background image with a background-size instead, because I don't think this works fine with high-dpi devices. I haven't tried yet though.